src/i_midimusic.c

```diff
@@ -1854,53 +1566,28 @@ static void I_WIN_UnRegisterSong(void *handle)
     song.rpg_loop = false;
 }
 
-static void I_WIN_ShutdownMusic(void)
+static void I_MID_ShutdownMusic(void)
 {
-    MMRESULT mmr;
-
-    if (!hMidiStream)
+    if (!music_initialized)
     {
         return;
     }
 
-    EnterCriticalSection(&CriticalSection);
-    StreamStop();
-    I_WIN_UnRegisterSong(NULL);
-    win_midi_state = STATE_SHUTDOWN;
-    StreamStart();
-    LeaveCriticalSection(&CriticalSection);
+    I_MID_StopSong(NULL);
+    I_MID_UnRegisterSong(NULL);
 
-    if (WaitForSingleObject(hPlayerThread, PLAYER_THREAD_WAIT_TIME) == WAIT_OBJECT_0)
-    {
-        CloseHandle(hPlayerThread);
-        hPlayerThread = NULL;
-    }
-    StreamStop();
+    // Send notes/sound off prior to reset to prevent volume spikes.
+    SendNotesSoundOff();
```

Comment:
`SendNotesSoundOff()` can be removed because it's called above in `I_MID_StopSong()`.

Reply:
Fixed in https://github.com/fabiangreffrath/woof/pull/1635/commits/ff394d25026903a543b5ad3fe7025b325861060f